### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by geckogecko**
added german translation
For questions: 'Georg Steinbacher' @ Ark Slack
My Ark-Wallet: AXjrTYRkmZ9EcsHDQ1kw2FJEBF8qUWo4qA

### CODE REVIEW

**Comment by Nasicus** [comment]
I'll review this, since my native language is also german :)

File: src/assets/i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  "SEARCH": "Suchen",
+  "SETTINGS_PAGE": {
+    "ABOUT": "Über",
+    "APPEARANCE": "Erscheinung",
+    "APPLICATION": "Applikation",
+    "CHANGE_PIN": "Ändere PIN",
+    "CLEAR_DATA": "Lösche Daten",
+    "CLEAR_DATA_TEXT": "Alle deine Daten, inklusive Profile, Wallets und Kontakte werden von der App gelöscht und zurückgesetzt.",
+    "CURRENCY": "Währung",
+    "DARK_MODE": "Dark Mode",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Nachtmodus

File: src/assets/i18n/de.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    "LANGUAGE": "Sprache",
+    "PRIVACY_POLICY": "Privatsphäre",
+    "SECURITY": "Sicherheit",
+    "SELECT_WALLET_FIRST_TEXT": "Wähle zuerst ein Wallet auf der Wallets Seite",
+    "NOT_AVAILABLE_WATCH_ONLY": "Nicht verfügbar für reine Leseadresse",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: `Leseadresse` = `Watch-Only Adresse`

File: src/assets/i18n/de.json [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    "SETTINGS": "Einstellungen",
+    "VERSION": "Version",
+    "WALLET_BACKUP": "Wallet Backup"
+  },
+  "SHOW_ADVANCED": "Zeige Fortgeschrittene",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Zeige Details

File: src/assets/i18n/de.json [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+  "SKIP": "Überspringen",
+  "STATUS": "Status",
+  "TRANSACTIONS_PAGE": {
+    "2ND_SIGNATURE_CREATION": "Zweite Signatur erstellen",
+    "ADD_ADDRESS_TO_CONTACTS": "Füge {{ address }} zu Kontakten hinzu",
+    "AMOUNT": "Menge",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Anzahl

File: src/assets/i18n/de.json [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+    "CONFIRMATIONS": "Bestätigungen",
+    "CONTACT_NAME": "Neuer Kontakt Name (optional)",
+    "COPY_TXID": "Kopiere TXID",
+    "CREATE_TRANSACTION_ERROR": "Konnte Transaktion nicht erstellen, checke die Internetverbindung",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Konnte Transaktion nicht erstellen, prüfe die Internetverbindung

File: src/assets/i18n/de.json [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+    "SEND_TRANSACTION": "Sende Transaktion",
+    "SENT_SUCCESS": "Transaktion erfolgreich gesendet!",
+    "SMART_BRIDGE": "SmartBridge (optional)",
+    "TO": "An",
+    "TRANSACTION": "Transaktion"
+  },
+  "TYPE": "Typ",
+  "WALLETS_PAGE": {
+    "ADD_WALLET_ERROR": "Konnte Wallet nicht hinzufügen",
+    "ADD_WALLET_TEXT": "Um Wallet zu importieren oder hinzuzufügen, klicke den nachfolgenden Button",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Um ein Wallet zu importieren oder hinzuzufügen, klicke den nachfolgenden Button

File: src/assets/i18n/de.json [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+    "ADDRESS": "Adresse",
+    "ALERT_SUCCESSFULLY_ENCRYPTED_SECOND_PASSPHRASE": "Die zweite Passphrase wurde erfolgreich verschlüsselt",
+    "BACKUP_ACCOUNT": "WICHTIG! Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen und ALLE DEINE ARK MÜNZEN WERDEN VERLOREN SEIN.",
+    "BACKUP_PASSPHRASE": "Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
+    "BACKUP_SECOND_PASSPHRASE": "Sichere diese zweite Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
+    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst ein Passwort deiner Wahl hinzufügen. Vergiss nicht dies zu sichern, du benötigst es für die Bestätigung.",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: I think `password` is wrong and should be `passphrase` (also in english version).

`Du kannst ein Passphrase deiner Wahl hinzufügen. Vergiss nicht diesen zu sichern, du musst ihn nachher nochmals zur Bestätigung eingeben.`

File: src/assets/i18n/de.json [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+    "BALANCE": "Saldo",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Kontostand

File: src/assets/i18n/de.json [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+    "BIP38_SEED": "BIP38 Seed",
+    "CREATE_WALLET": "Erstelle Wallet",
+    "ENTER_SECRET_PASSPHRASE": "Gib deine geheime Passphrase ein",
+    "ENTROPY": "Entropy",
+    "ENTROPY_RANDOM_BYTES": "Wische, um zufällige Bytes zu erzeugen",
+    "GENERATE_ENTROPY": "Erzeuge Entropy",
+    "IMPORT_INVALID_ADDRESS": "Diese Address kann nicht in das Profil importiert werden",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: typo: `Adresse`

File: src/assets/i18n/de.json [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+    "IMPORT_WALLET": "Importiere Wallet",
+    "KEEP_SAFE": "Sichere es",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: `Bewahre es sicher auf!`

File: src/assets/i18n/de.json [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+    "LABEL": "Marke",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Bezeichnung

File: src/assets/i18n/de.json [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+    "MY_WALLET": "Mein Wallet",
+    "NEW_WALLET_NAME": "Neuer Wallet Name",
+    "NO_TRANSACTIONS": "Keine Transaktionen verfügbar",
+    "PASSPHRASE": "Passphrase",
+    "PASSPHRASE_NOT_MATCH": "Die Passphrase passt nicht zu der die du gerade erhalten hast.",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Die Passphrase passt nicht zu der, die du gerade erhalten hast.

File: src/assets/i18n/de.json [hunk 13]
```diff
@@ -0,0 +1,237 @@
...
+    "PUBLIC_ADDRESS": "Öffentliche Adresse",
+    "PUBLIC_ADDRESS_SHARE": "Öffentliche Adress (TEILE)",
+    "PUBLIC_KEY": "Öffentlicher Schlüssel",
+    "RE_ENTER_PASSPHRASE": "Wiederhole deine Passphrase",
+    "REMOVE_WALLET": "Lösche Wallet",
+    "REMOVE_WALLET_TEXT": "Dein Konto wird gelöscht. Stelle sicher, dass du ein Backup deiner Passphrase hast, du brauchst diese um dein Konto wiederherzustellen.",
+    "SCAN_QRCODE": "Scanne QR Code",
+    "SCAN_QRCODE_TEXT": "Importiere deine Adresse oder Passphrase mittels QR Code, einfach scannen!",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: `Importiere deine Adresse oder Passphrase mittels scannen eines QR Codes`.

File: src/assets/i18n/de.json [hunk 14]
```diff
@@ -0,0 +1,237 @@
...
+    "SECOND_BIP38_SEED": "Zweiter BIP38 Seed",
+    "SECOND_PASSPHRASE": "Zweite Passphrase",
+    "SECOND_WIF": "Zweiter WIF",
+    "SECRET_PASSPHRASE": "Geheime Passphrase",
+    "SECRET_PASSPHRASE_KEEP_SAFE": "Geheime Passphrase (GEHEIM HALTEN)",
+    "SEED": "Seed",
+    "TOTAL_BALANCE": "Gesamtsumme",
+    "USERNAME": "Username",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Benutzername

File: src/assets/i18n/de.json [hunk 15]
```diff
@@ -0,0 +1,237 @@
...
+    "WALLETS": "Wallets",
+    "WIF": "WIF"
+  },
+  "WATCH_ONLY": "Nur Anzeigen",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Watch-Only

File: src/assets/i18n/de.json [hunk 16]
```diff
@@ -0,0 +1,237 @@
+{
+  "API": {
+    "BALANCE_TOO_LOW": "Saldo zu gering für Transaktion",
```
[the remark this answers is not in the document]
Comment (geckogecko) [reply src/assets/i18n/de.json]: I used 'Saldo' because 'Kontostand' somehow feels wrong for me in the context of crypto currencies

File: src/assets/i18n/de.json [hunk 17]
```diff
@@ -0,0 +1,237 @@
...
+    "MISSED_BLOCKS": "Verpasste Blöcke",
+    "NO_MATCHING_SEARCH": "Keine passenden Delegates",
+    "NOTE_COST_TEXT": "Beachte, die Kosten der Registrierung eines Delegates sind",
+    "PRODUCED_BLOCKS": "Produzierte Blöcke",
+    "RANK": "Rang",
+    "REGISTER_DELEGATE": "Registriere Delegate",
+    "STANDBY": "Standby",
+    "SUPPLY": "Bereitstellen",
+    "TOTAL_FORGED": "Gesamt Forged",
+    "UNVOTE": "Abwählen",
```
[the remark this answers is not in the document]
Comment (geckogecko) [reply src/assets/i18n/de.json]: hard to say. I think vote/unvote should stay consistent -> 'Stimme hinzufügen' and 'Stimme entfernen' or 'Wählen' and 'Abwählen'

File: src/assets/i18n/de.json [hunk 18]
```diff
@@ -0,0 +1,237 @@
...
+    "SECURITY": "Sicherheit",
+    "TEXT_1": "Wir bieten alle Werkzeuge um deine ARK Münzen zu verwalten, jederzeit, überall, weltweit!",
+    "TEXT_2": "All deine vertraulichen Daten sind sicher gespeichert mittels PBKDF2 Algorithmus und AES-256 Verschlüsselung geschützt mit deinem PIN Code",
+    "TEXT_3": "Wechsle zwischen Profilen und Wallets, check deinen Saldo, sende und empfange Guthaben und vieles mehr",
+    "WELCOME": "Willkommen an Board"
+  },
+  "LOGIN_PAGE": {
+    "ARK_WALLET": "ARK Wallet",
+    "CREATE_PROFILE": "Erstelle ein Profil",
+    "SLOGAN": "Wilkommen zum ARK Ecosystem"
```
[the remark this answers is not in the document]
Comment (geckogecko) [reply src/assets/i18n/de.json]: same

File: src/assets/i18n/de.json (hunk 2 above)
Comment (geckogecko) [reply src/assets/i18n/de.json]: ah ok -> was totaly unsure if I should translate 'Watch-Only' to german or not

**Comment by geckogecko** [comment]
@Nasicus thanks for the review. I agree with all your comments with no answer from myself.

**Comment by alexbarnsley** [comment]
Let me know once they're all changed and you're both happy to merge the PR

**Comment by Nasicus** [comment]
@geckogecko I  don't know if you know this, but I cannot edit your branch / changes. So you have to implement my suggestions your self :)

**Comment by geckogecko** [comment]
@alexbarnsley I fixed the review comments
@Nasicus are you happy with them ?

File: src/assets/i18n/de.json [hunk 19]
```diff
@@ -181,49 +181,49 @@
...
-    "ADD_WALLET_TEXT": "Um Wallet zu importieren oder hinzuzufügen, klicke den nachfolgenden Button",
+    "ADD_WALLET_TEXT": "Um ein Wallet zu importieren oder hinzuzufügen, klicke den nachfolgenden Button",
     "ADDRESS": "Adresse",
     "ALERT_SUCCESSFULLY_ENCRYPTED_SECOND_PASSPHRASE": "Die zweite Passphrase wurde erfolgreich verschlüsselt",
     "BACKUP_ACCOUNT": "WICHTIG! Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen und ALLE DEINE ARK MÜNZEN WERDEN VERLOREN SEIN.",
     "BACKUP_PASSPHRASE": "Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
     "BACKUP_SECOND_PASSPHRASE": "Sichere diese zweite Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
-    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst ein Passwort deiner Wahl hinzufügen. Vergiss nicht dies zu sichern, du benötigst es für die Bestätigung.",
+    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst ein Passphrase deiner Wahl hinzufügen. Vergiss nicht diesen zu sichern, du musst ihn nachher nochmals zur Bestätigung eingeben.",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Passphrase always used as "female" except for here => change too!

File: src/assets/i18n/de.json (hunk 8 above)
Comment (Nasicus) [reply src/assets/i18n/de.json]: Why not changed? Do you think `Saldo` is better?

File: src/assets/i18n/de.json (hunk 10 above)
Comment (Nasicus) [reply src/assets/i18n/de.json]: Ignored on purpose?

File: src/assets/i18n/de.json [hunk 20]
```diff
@@ -181,49 +181,49 @@
...
     "REMOVE_WALLET": "Lösche Wallet",
     "REMOVE_WALLET_TEXT": "Dein Konto wird gelöscht. Stelle sicher, dass du ein Backup deiner Passphrase hast, du brauchst diese um dein Konto wiederherzustellen.",
     "SCAN_QRCODE": "Scanne QR Code",
-    "SCAN_QRCODE_TEXT": "Importiere deine Adresse oder Passphrase mittels QR Code, einfach scannen!",
+    "SCAN_QRCODE_TEXT": "Importiere deine Adresse oder Passphrase mittels scannen eines QR Codes",
     "SECOND_BIP38_SEED": "Zweiter BIP38 Seed",
     "SECOND_PASSPHRASE": "Zweite Passphrase",
     "SECOND_WIF": "Zweiter WIF",
     "SECRET_PASSPHRASE": "Geheime Passphrase",
-    "SECRET_PASSPHRASE_KEEP_SAFE": "Geheime Passphrase (GEHEIM HALTEN)",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Why did you change this? I thought this was fine :)

File: config.xml [hunk 21]
```diff
@@ -114,4 +111,7 @@
     <plugin name="cordova-plugin-x-socialsharing" spec="^5.2.1" />
     <plugin name="cordova-sqlite-storage" spec="^2.1.4" />
     <plugin name="ionic-plugin-keyboard" spec="^2.2.1" />
+    <engine name="android" spec="^6.4.0" />
+    <engine name="browser" spec="^5.0.1" />
+    <engine name="ios" spec="^4.5.4" />
 </widget>
```
[the remark this answers is not in the document]
Comment (Nasicus) [reply config.xml]: This probably got forgotten to - just revert the changes you did in this file :)

File: src/assets/i18n/de.json (hunk 16 above)
[the remark this answers is not in the document]
Comment (Nasicus) [reply src/assets/i18n/de.json]: Ok - no problem - let's keep it like that

File: src/assets/i18n/de.json [hunk 22]
```diff
@@ -0,0 +1,237 @@
...
+    "WELCOME": "Willkommen an Board"
+  },
+  "LOGIN_PAGE": {
+    "ARK_WALLET": "ARK Wallet",
+    "CREATE_PROFILE": "Erstelle ein Profil",
+    "SLOGAN": "Wilkommen zum ARK Ecosystem"
+  },
+  "LOGOUT_PROFILE_TEXT": "Ausloggen?",
+  "MARKETS_PAGE": {
+    "MARKET_CAP": "Markt Cap.",
```
[the remark this answers is not in the document]
Comment (Nasicus) [reply src/assets/i18n/de.json]: Thought about this again, I'd either use `Marktkapitalisierung ` or `Market Cap`

File: src/assets/i18n/de.json (hunk 18 above)
[the remark this answers is not in the document]
Comment (Nasicus) [reply src/assets/i18n/de.json]: I think `beim` is probably better

**Comment by geckogecko** [comment]
@Nasicus updated

File: src/assets/i18n/de.json [hunk 23]
```diff
@@ -187,7 +187,7 @@
     "BACKUP_ACCOUNT": "WICHTIG! Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen und ALLE DEINE ARK MÜNZEN WERDEN VERLOREN SEIN.",
     "BACKUP_PASSPHRASE": "Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
     "BACKUP_SECOND_PASSPHRASE": "Sichere diese zweite Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
-    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst ein Passphrase deiner Wahl hinzufügen. Vergiss nicht diesen zu sichern, du musst ihn nachher nochmals zur Bestätigung eingeben.",
+    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst eine Passphrase deiner Wahl hinzufügen. Vergiss nicht diesen zu sichern, du musst ihn nachher nochmals zur Bestätigung eingeben.",
```
Comment (Nasicus) [inline src/assets/i18n/de.json]: Sorry for picking this up again, you forgot to change the second part of the sentence:

`Du kannst eine Passphrase deiner Wahl hinzufügen. Vergiss nicht diese zu sichern, du musst sie nachher nochmals zur Bestätigung eingeben.`

**Comment by alexbarnsley** [comment]
@Nasicus is that above the last translation you think needs doing?

**Comment by Nasicus** [comment]
I think so.

**Comment by alexbarnsley** [comment]
Thanks @Nasicus 

@geckogecko would you mind just making the last translation(s) above please. We can try and get this merged in today then

**Comment by geckogecko** [comment]
Hi sry I forgot about this last issue

**Comment by alexbarnsley** [comment]
No worries 😁

**Comment by boldninja** [comment]
@geckogecko sent, thank you for your contribution.
